ang/civicase/ActivityFeed.js

```diff
@@ -419,30 +419,41 @@
         $activityDetailsPanel = $element.find('.civicase__activity-panel');
         activityPanelMeasurements = ActivityPanelMeasurements($activityDetailsPanel);
 
-        affixActivityDetailsPanel();
+        setActivityDetailsPanelAffixOffsets();
         $rootScope.$on('civicase::case-search::dropdown-toggle', resetAffix);
       }());
 
       /**
        * Sets Activity Details Panel affix offsets
        */
-      function affixActivityDetailsPanel () {
+      function setActivityDetailsPanelAffixOffsets () {
         $activityDetailsPanel.affix({
           offset: {
             top: activityPanelMeasurements.getTopOffset(),
             bottom: activityPanelMeasurements.getBottomOffset()
           }
         });
 
-        $activityDetailsPanel.on('affixed.bs.affix', function () {
-          $activityDetailsPanel
-            .css('top', activityPanelMeasurements.getDistanceFromTop())
-            .width($element.width());
-        }).on('affixed-top.bs.affix', function () {
-          $activityDetailsPanel
-            .css('top', 'auto')
-            .css('width', 'auto');
-        });
+        $activityDetailsPanel
+          .on('affixed.bs.affix', setActivityDetailsPanelPosition)
+          .on('affixed-top.bs.affix', function () {
+            $activityDetailsPanel
+              .css('top', 'auto')
+              .css('width', 'auto');
+          });
```

Comment:
Now that I take a look at this I think we should move these event listeners to a `initWatchers` functions or similar.